ui/app/src/components/ErrorDialog/ErrorDialog.tsx

```diff
@@ -20,13 +20,14 @@ import CloseIcon from '@mui/icons-material/CloseRounded';
 import Dialog from '@mui/material/Dialog';
 import StandardAction from '../../models/StandardAction';
 import { ApiResponse } from '../../models/ApiResponse';
-import ApiResponseErrorState from '../ApiResponseErrorState';
+import ApiResponseErrorState, { ApiResponseErrorStateProps } from '../ApiResponseErrorState';
```

Comment:
Please configure your IDE to add the `type` annotation on imports.
```suggestion
import ApiResponseErrorState, { type ApiResponseErrorStateProps } from '../ApiResponseErrorState';
```

Reply:
Done, IDE is now configured to add the type annotations, thanks!